week1.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Sat Oct  5 10:50:23 2019
+
+@author: vivek
+"""
+
+import pandas as pd
+import psycopg2
+
+employees = pd.read_csv("~/Desktop/DataWarehousing/HW1/employees.csv")
+orders = pd.read_csv("~/Desktop/DataWarehousing/HW1/orders.csv")
+products = pd.read_csv("~/Desktop/DataWarehousing/HW1/products.csv")
+
+conn = psycopg2.connect("dbname=week1 user=postgres host=localhost")
+cursor = conn.cursor()
+
+office_info_df = employees.loc[:,['office_code','city', 'state', 'country', 'office_location']].drop_duplicates()
+tuples = [tuple(x) for x in office_info_df.values]
+for i in tuples:
+    cursor.execute("INSERT INTO office_info (office_code, city, state, country, office_location) VALUES(%s, %s, %s, %s, %s)", i)
+
+employee_info_df = employees.loc[:,['employee_number', 'office_code', 'last_name', 'first_name', 'reports_to', 'job_title']].drop_duplicates()
```

Comment:
Looks great, but you should make sure to deal with NULL values, are they properly deal with here?